### ISSUE TITLE
Feature: Support for mixed 3d/2d systems.

### ISSUE THREAD

**Issue by WillBaldwin0** [WillBaldwin0/LDHP-builder#2]
At the moment, only pure 2d systems are supported. i.e., one layer of corner sharing octahedra. 

Extending to have some support for mixed systems is easy, and it just remains to be seen whether the search space or molecule orientations is small enough to make this useful in practice.

**Comment by zyyyfd** [comment]
Hello, I would like to extend LDHP to 2d hybrid perovskite. The current plan is to fix the three organic molecules MA,FA and Cs into the inorganic layer and add layer number parameters. May I ask whether this scheme is reasonable? Thank you for your answer.

**Comment by WillBaldwin0** [comment]
Yes I think this should be straightforward. If you want to develop this on a fork, and then make a PR, I would be happy to advise. I think the code is quite easy to understand, and so you should be able to make these modifications relatively easily.

**Comment by zyyyfd** [comment]
Hello, I have been in your code added on the basis of constructing the function of the quasi two dimensional perovskite, currently stored in https://github.com/zyyyfd/LDHP-builder-quasi-2D, please have a look.
